### PR TITLE
Doc write to remote

### LINKED ISSUE #6096: add other to_parquet examples
It took me longer than I would have liked to work out how to save a `dask.dataframe` in an Azure blob.

https://stackoverflow.com/questions/60765331/moving-data-from-a-database-to-azure-blob-storage/60773724#60773724

Is there any appetite for adding another example to https://github.com/dask/dask/blob/master/dask/dataframe/io/parquet/core.py#L337?

The example there currently is about as minimal as you can get (which I understand is the idea).

Or there could be a separate page/section for this. i.e. near https://docs.dask.org/en/latest/dataframe-create.html#locations

### PR DESCRIPTION
**Pull request by raybellwaves**
Closes #6096 

- [NA] Tests added / passed
- [NA] Passes `black dask` / `flake8 dask`

Tested doc build

image

### CODE REVIEW

File: docs/source/dataframe-create.rst [hunk 1]
```diff
@@ -50,8 +50,26 @@ Pandas:
 .. autosummary::
     from_pandas
 
-Locations
----------
+Writing to remote locations
```
Comment (TomAugspurger) [inline docs/source/dataframe-create.rst:53]: I think reading should come before writing. It's probably more common.

Comment (TomAugspurger) [reply docs/source/dataframe-create.rst:53]: Reading through the document, I'd prefer to have all the "creation" methods first, and then some discussion of writing methods afterwords.

**Comment by raybellwaves** [comment]
thanks for the feedback @TomAugspurger.

You thinking of a "Creation" section after "Reading from various locations" and before "Writing to remote locations"?

By "Creation" do you mean similar to the example I gave? i.e. `from_pandas`
```
>>> d = {'col1': [1, 2, 3, 4], 'col2': [5, 6, 7, 8]}
>>> df = dd.from_pandas(pd.DataFrame(data=d), npartitions=2)
```
or are you thinking more `to_csv` and `to_parquet`?

**Comment by TomAugspurger** [comment]
Something like

```
- Creation
  - Locations
  - Dask Delayed
  - From Raw Dask Graphs
- Storing
  - Locations
```

i.e. group by topic first (creating vs. storing) and then split those by method.

**Comment by jcrist** [comment]
Thanks @raybellwaves. I've updated the docs formatting - if you want hierarchical headers in rst files (i.e. `Creating` above `Reading from various locations`), you need to select a different underline character for each level (the actual character doesn't matter, as long as it's consistent in the documentation). Will merge on tests pass.

**Comment by raybellwaves** [comment]
Thanks @jcrist

**Pull request merged**